tools/tab_checker

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+pr_id=$1
+files=$(curl https://api.github.com/repos/os-autoinst/scripts/pulls/${pr_id}/files | jq -c '.[].filename' | sed 's/\"//g')
```

Comment:
You probably want to silent the curl and get rid of the `sed`, because that will delete also quotes in the middle.
```suggestion
files=$(curl -s https://api.github.com/repos/os-autoinst/scripts/pulls/${pr_id}/files | jq -r -c '.[].filename')
```